### PR TITLE
Add master configuration for scheduleable/unschedueable.

### PR DESCRIPTION
**Pull request by aizuddin85**
Add how to make master to be scheduleable or unscheduleable.

### CODE REVIEW

File: content/configuration/_index.md [hunk 1]
```diff
@@ -51,3 +51,11 @@ Or:
 ```
 oc exec -n openshift-sdn $(oc get pods -n openshift-sdn -l app=sdn --no-headers=true -o custom-columns=:metadata.name|head -n1) cat /config/{kube-proxy-config,sdn-config}.yaml
 ```
+
+# Making Master Unscheduleable/Scheduleable
+To configure master as unscheduleable (when UPI installation without setting this prior to the install):
+```
+#oc patch --type=merge --patch='{"spec":{"mastersSchedulable": false}}' schedulers.config.openshift.io cluster
```
Comment (e-minguez) [inline content/configuration/_index.md]: Can you remove the `#` to be consistent with the rest of the tips? Thanks!

File: content/configuration/_index.md [hunk 2]
```diff
@@ -51,3 +51,11 @@ Or:
 ```
 oc exec -n openshift-sdn $(oc get pods -n openshift-sdn -l app=sdn --no-headers=true -o custom-columns=:metadata.name|head -n1) cat /config/{kube-proxy-config,sdn-config}.yaml
 ```
+
+# Making Master Unscheduleable/Scheduleable
+To configure master as unscheduleable (when UPI installation without setting this prior to the install):
+```
```
Comment (e-minguez) [inline content/configuration/_index.md]: Can you add a new line between the content and the code? Thanks!

Comment (aizuddin85) [reply content/configuration/_index.md]: fixed this

File: content/configuration/_index.md (hunk 1 above)
Comment (aizuddin85) [reply content/configuration/_index.md]: fixed this as well.

**Comment by e-minguez** [comment]
Thanks for the PR! Merged

**Pull request merged**